src/vscode-dts/vscode.d.ts

```diff
@@ -7505,7 +7506,7 @@ declare module 'vscode' {
 		 * must be activated. Check whether {@link TerminalShellExecution.exitCode} is rejected to
 		 * verify whether it was successful.
 		 *
-		 * @param command A command to run.
+		 * @param executable A command to run.
```

Comment:
Not sure why these weren't caught before. All seem like real lint errors that should have been reported